### ISSUE TITLE
moment dependency

### ISSUE THREAD

**Issue by teug91** [dhis2/ui-widgets#9]
`es/locales/index.js` imports `moment`, but it is not listed as a dependency. Getting the below error in an app which doesn't use moment.
```
../node_modules/@dhis2/ui-widgets/build/es/locales/index.js
Module not found: Can't resolve 'moment' in '../node_modules/@dhis2/ui-widgets/build/es/locales'
```

**Comment by teug91** [comment]
[Repo](https://github.com/dhis2designlab/app-base)

**Comment by varl** [comment]
Yeah, that `locales/index.js` is autogenerated and assumes that moment should be a dependency.

We're not fans of that at all since this forces the entire thing to be bundled. It's being reworked, but can be added as a dependency of your app as a fast workaround.

Another way is to explicitly set it as a dependency of d2-i18n which makes the assumption that it exists. The actual culprit is https://github.com/dhis2/d2-i18n-generate/ which is a devDependency, but states moment as a runtime dependency.

Adding the dep to d2-i18n as a runtime dep should be sufficient as well:

https://github.com/dhis2/d2-i18n/pull/7

**Comment by teug91** [comment]
Thanks! Just wanted to make sure you were aware.

**Comment by varl** [comment]
Cool. Thanks for taking the time to report it, @teug91. I've published **@dhis2/d2-i18n** version **1.0.5** which has the explicit dependency set. The dependabot should pick up the dep change soon and I'll merge it to @dhis2/ui-widgets ASAP.
